src/main.js

```diff
@@ -11,14 +11,31 @@ const context = github.context;
 async function run() {
   try {
     const { owner, repo } = context.repo;
-    const supportEventNames = ['issues', 'issue_comment', 'pull_request', 'pull_request_target'];
+    const supportEventNames = [
+      'issues',
+      'issue_comment',
+      'pull_request',
+      'pull_request_target',
+      'release',
+    ];
     if (supportEventNames.includes(context.eventName)) {
       const require = core.getInput('require');
 
       const isIssue = context.eventName.startsWith('issue');
-      const username = isIssue
-        ? context.payload.issue.user.login
-        : context.payload.pull_request.user.login;
+      const isRelease = context.payload.release !== undefined;
+      let username = undefined;
+
+      if (isRelease) {
+        username = context.payload.release.author.login;
+      } else if (isIssue) {
+        username = context.payload.issue.user.login;
+      } else {
+        username = context.payload.pull_request.user.login;
+      }
```

Comment:
I just realized we have access to `github.actor`
Not sure we should really check against all of those
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

Reply:
Any insights?

Reply:
Added a new commit, I think it actually make sense.